docker/provision/provision_entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+sleep 10
+python3 /home/provision/girder.py
+cd /home/data && python3 ingest.py ./data girder 8080 admin letmein
+cd /home/data && python3 ingest_1000.py ./data girder 8080 admin letmein
```

Comment:
We probably don't need to provision each time we start the container, right? Maybe just leave them out and run them manually?

Reply:
The scripts are idempotent so running it multiple times is quick and yields the same result. I wanted to do everything as part of docker deployment so that we do not have to run manual steps. I don't have a preference this just seemed easier for deployment :).

Reply:
The data ingestion is kind of two heavy as part of the startup. I am thinking that data is not part of application. I think we will have more data, we don't want to always copy them to the docker image. The beauty of girder client-based data ingestion is that we can run it from anywhere and the original data don't have to be exists on the server.

Reply:
Ok, I can make that change so that ingest scripts are not part of provisioning but a post provisioning step that we do manually. I will make that change soon.

Reply:
Cool! and all the parameterize work is useful either way.

Reply:
@matthewma7 I addressed your comment. PTAL

Reply:
Cool. Thanks!

Reply:
We could merge this one now.